website/versioned_docs/version-0.2.0/getting-setup.md

```diff
@@ -0,0 +1,121 @@
+# Getting Setup With Bitcoin-S
```

Comment:
This is not in the right place, you need to place this inside of `docs/` folder. You may want to replace the `getting-started.md` file there? 

For more infromation on how the website works see: 

https://bitcoin-s.org/docs/next/contributing-website

If you are not going to replace the existing file, you need to update `sidebars.json` to how this new file. 

See this PR: https://github.com/bitcoin-s/bitcoin-s/pull/948